meteor-app/imports/ui/reducers/initialState.js

```diff
@@ -14,23 +14,37 @@ export default {
     //! For testing only. Should be empty on start.
     layers: [
       {
-        name: 'Water-year Precipitation',
-        urlTile: 'PPT_water_year',
```

Comment:
The `urlTile` is used as ID for the layer, while the `name` is purely used for displaying purposes. Please don't mix them together.